From: Sophie Brun <sophie@offensive-security.com>
Date: Tue, 2 Feb 2021 09:54:06 +0100
Subject: UPdate shebang to indicate python3

---
 metagoofil.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metagoofil.py b/metagoofil.py
index 3ddffaf..cb556d2 100644
--- a/metagoofil.py
+++ b/metagoofil.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Standard Python libraries.
 import argparse
